### ISSUE TITLE
How can I inject a Chrome Extension. Its been 4 years.. Is this seriously not implemented yet?

### ISSUE THREAD

**Issue by skylerspark** [jiahaog/nativefier#985]
- [x] I looked at `nativefier --help` and https://github.com/jiahaog/nativefier/blob/master/docs/api.md , no existing option fits my needs.
- [x] I searched existing open & closed issues. No, my issue is NOT new.

**Suggested solution**
Electron has supported injecting dev extensions for a long time now. I want to be able to inject my pandora enchancer plugin so I dont get ads in pandora

**Alternative solutions**
I know I can inject javascript, but I dont have the code for the plugin or the ad remover in this specific scenario.

**Context**
 - Nativefier: 8.0.6
 - Node.js: 12.16.1
 - Npm: 6.13.4
 - OS: Windows 10 Pro Version 10.0.18363 Build 18363
 - Additional context: (for example: "I'm behind a proxy, with configuration X and protocol Y")

Basically, this has existed for a long time in electron. We should be able to do this in nativefier by now?

**Comment by ronjouch** [comment]
https://github.com/jiahaog/nativefier/issues/207

**Comment by Rimseg** [comment]
@MisterSirCode 

> We should be able to do this in nativefier by now?

Yes, you can do a workaround:
https://github.com/jiahaog/nativefier/issues/246#issuecomment-718708717
